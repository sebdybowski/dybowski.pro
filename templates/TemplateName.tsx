import React, { FC } from 'react';

import styles from './TemplateName.module.scss';

interface TemplateNameProps {}

const TemplateName: FC<TemplateNameProps> = () => (
  <div className={styles.TemplateName} data-testid="TemplateName">
    TemplateName Component
  </div>
);

export default TemplateName;
