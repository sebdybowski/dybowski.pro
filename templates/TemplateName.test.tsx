import React from 'react';
import { render } from '@testing-library/react';

import TemplateName from './TemplateName';

describe('<TemplateName />', () => {
  test('it should mount', () => {
    const { asFragment } = render(<TemplateName />);
    
    expect(asFragment()).toMatchSnapshot();
  });
});
