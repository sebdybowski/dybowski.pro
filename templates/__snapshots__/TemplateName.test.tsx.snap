// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TemplateName /> it should mount 1`] = `
<DocumentFragment>
  <div
    class="Hero"
    data-testid="TemplateName"
  >
    TemplateName Component
  </div>
</DocumentFragment>
`;
